includes/admin/class-rop-global-settings.php

```diff
@@ -386,7 +386,11 @@ public static function instance() {
 				self::$instance->services_defaults
 			);
 
-			self::$instance->settings_defaults['min_interval'] = apply_filters( 'rop_min_interval_bw_shares_min', ROP_DEBUG ? 0.05 : 0.5 );
+			$min_interval = 0.5;
+			if ( empty( get_option( 'rop_data', '' ) ) ) {
+				$min_interval = self::$instance->license_type() > 0 ? 0.5 : 12;
+			}
```

Comment:
NIT> You can combine them like (including the comment)
```php
// Apply new limit for new free users.
if ( empty( get_option( 'rop_data', '' ) ) && 1 < self::$instance->license_type() ) { 
    $min_interval = 12;
}

Reply:
Done! Please review my latest commit.